module/skill/GetUp/data/config/GetUp.yaml

```diff
@@ -0,0 +1,22 @@
+# Controls the minimum log level that NUClear log will display
+log_level: DEBUG
+
+scripts:
+  getup_upright:
+   - "Stand.yaml"
```

Comment:
this seems strange

Reply:
If you emit a getup task and the robot is upright I'd say the correct thing to do is to stand?
Any other getup script we run will result in the robot flinging itself to the ground.

Reply:
can we not just do nothing and emit done?